src/TypographyCatalog.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ComponentCatalogPanel from './ComponentCatalogPanel.js';
+
+const TypographyCatalog = () => {
+  return (
+    <ComponentCatalogPanel
+      hero={<TypographyHero/>}
+      title='Typography'
+      description='Roboto and Noto are the standard typefaces on Android and Chrome.'
```

Comment:
Drop `Noto` from the description. We don't provide anything for it, and it could people about what we're showing on this page.